Formula/helmfile.rb

```diff
@@ -0,0 +1,30 @@
+class Helmfile < Formula
+  desc "Deploy Kubernetes Helm Charts"
+  homepage "https://github.com/roboll/helmfile"
+  url "https://github.com/roboll/helmfile/archive/v0.18.0.tar.gz"
+  sha256 "d825db03b29dbe4842996a57a0751f67e61bfcf6350616eaa55caddbc515f7cf"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-helm"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/roboll/helmfile").install buildpath.children
+    cd "src/github.com/roboll/helmfile" do
+      system "go", "build", "-ldflags", "-X main.Version=v#{version}",
+             "-o", bin/"helmfile", "-v", "github.com/roboll/helmfile"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    (testpath/"helmfile.yaml").write <<-EOS
+      repositories:
+      - name: stable
+        url: https://kubernetes-charts.storage.googleapis.com/
+    EOS
+    system "helm", "init", "--client-only"
+    output = /"stable" has been added to your repositories/
```

Comment:
The regex use without actually checking a regex is kind of upsetting, but I'm not sure:
```ruby
output = "\"stable\" has been added to your repositories"
```
is especially stylistically pleasing either 😭.

Reply:
I swapped to single quotes expecting to see a style error, but this passed:

```
brew test helmfile
brew audit --new-formula helmfile
brew audit --strict --online helmfile
brew style helmfile
```

Is there an audit/style check I missed, or is single-quotes good? I'm assuming it's good, so have pushed that change.